### PR TITLE
Create Today's actual date

### PR DESCRIPTION
**Pull request by aiotv1**
Create Today's actual date
Example :
monday , december , 2023 ,24

### CODE REVIEW

File: extensions/Today's actual date [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class DateTimeExtension {
```
Comment (LilyMakesThings) [inline extensions/Today's actual date:1]: Extensions that are submitted must be wrapped in an [immediately-invoked-function-expression (IIFE)](https://docs.turbowarp.org/development/extensions/unsandboxed#:~:text=The%20unsandboxed%20version%20would%20have%20code%20like%20this%3A).

File: extensions/Today's actual date [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  getInfo() {
+    return {
+      id: 'datetimeexample',
```
Comment (LilyMakesThings) [inline extensions/Today's actual date:4]: You extension ID must include your name in some form.

File: extensions/Today's actual date
Comment (LilyMakesThings) [inline extensions/Today's actual date:1]: Your extension file must be placed inside a folder with your name and be given the .js file extension.

**Review by LilyMakesThings — CHANGES_REQUESTED** [review]
Based on the mistakes made, you have clearly not read the [Developer Documentation](https://docs.turbowarp.org/development/extensions/introduction) and the [Contributing Guidelines](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md) **in full**.

I strongly advise you read those 2 documents before submitting an extension. Particularly pay close attention to [this].(https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md#acceptance-criteria)

**Comment by GarboMuffin** [comment]
see above feedback; please also refrain from relying on ChatGPT too heavily as it doesn't know that much about writing extensions

**Comment by GarboMuffin** [comment]
I will note that this is easy to port to normal Scratch scripts very directly

**Pull request closed without merging**